specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -46,6 +46,24 @@ suppressions:
     reason: Keep identical to stable GA version to avoid breaking changes. https://dev.azure.com/msdata/Vienna/_workitems/edit/2803196
     where:
       - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/registries/{registryName}/datareferences/{name}/versions/{version}"].post
+  - code: ProvisioningStateSpecifiedForLROPut
+    reason: Service already using 202 response code for the below APIs, got exceptions from ARM reviewer.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/endpoints/{endpointName}"].put
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/endpoints/{endpointName}/deployments/{deploymentName}"].put
+  - code: PutResponseCodes
+    reason: Service already using 202 response code for the below APIs, got exceptions from ARM reviewer.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/endpoints/{endpointName}"].put
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/endpoints/{endpointName}/deployments/{deploymentName}"].put
+  - code: AvoidAdditionalProperties
+    reason: This is an external reference right now, we will have full control on the schema returned in the upcoming version and will avoid this.
+    where:
+      - $.definitions.EndpointModels.properties.value.items
```

Comment:
I am ok to approve these suppressions , but this particular suppression is not working for AvoidAdditionalProperties, 
the linter is still complaining about this 

Can you please check once and follow the troubleshooting steps here : [https://eng.ms/docs/products/azure-developer-experience/design/specs-pr-guides/pr-suppressions](https://eng.ms/docs/products/azure-developer-experience/design/specs-pr-guides/pr-suppressions)

Reply:
The suppression seems not working even when I locate the "where"  to the $ref line, I previously try to make the scope larger on the whole definition level which is also not working, can you let me know if I missed anything?